database/rrdhost.c

```diff
@@ -308,7 +308,8 @@ RRDHOST *rrdhost_create(const char *hostname,
     }
 
     if (likely(!uuid_parse(host->machine_guid, host->host_uuid))) {
-        int rc = sql_store_host(&host->host_uuid, hostname, registry_hostname, update_every, os, timezone, tags);
+        int rc = sql_store_host(&host->host_uuid, hostname, registry_hostname, update_every, os, timezone, tags,
+                                host->system_info->hops);
```

Comment:
Hey Stelio, for unittests, system_info is NULL, so when running them this fails.

Reply:
Changed so that unit tests will ignore migration for now. Also changed so that sqlite unittest will run in an in memory sqlite database. So now the migration process needs additional unit tests to be added in another PR